core/metrics-core-impl/src/main/java/org/hawkular/metrics/core/impl/MetricsServiceImpl.java

```diff
@@ -227,6 +231,30 @@ void unloadDataRetentions() {
         dataRetentions.clear();
     }
 
+    private void initSystemTenant() {
+        CountDownLatch latch = new CountDownLatch(1);
+        Trigger trigger = new RepeatingTrigger.Builder().withInterval(30, MINUTES).withDelay(30, MINUTES).build();
+        dataAccess.insertTenant(new Tenant(SYSTEM_TENANT_ID))
+                .filter(ResultSet::wasApplied)
+                .map(row -> taskScheduler.scheduleTask(CreateTenants.TASK_NAME, SYSTEM_TENANT_ID, 100, emptyMap(),
```

Comment:
OK. I'm wondering if the system tenant shouldn't be created after schema 
setup and then we would simply fail the deployment if it no longer 
exists. After all, something very bad must have happened if the system 
tenant is no longer present.